src/Datadog.Trace.DuckTyping/DuckType.Statics.cs

```diff
@@ -38,5 +36,34 @@ public static partial class DuckType
         private static ModuleBuilder _moduleBuilder = null;
         [DebuggerBrowsable(DebuggerBrowsableState.Never)]
         private static AssemblyBuilder _assemblyBuilder = null;
+
+        /// <summary>
+        /// DynamicMethods delegates cache
+        /// </summary>
+        /// <typeparam name="TProxyDelegate">Proxy delegate type</typeparam>
+        public static class DelegateCache<TProxyDelegate>
+            where TProxyDelegate : Delegate
+        {
+            private static TProxyDelegate @delegate;
```

Comment:
NIT: Isn't the usual naming convention `_delegate` ?

Reply:
you're right, fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1054/commits/c95e340771917d62b1818ec1a6d8a2ce4a59fbff